ntoskrnl/io/pnpio.h

```diff
@@ -0,0 +1,54 @@
+#ifndef _PNPIO_H
```

Comment:
This file should probably live in `ntoskrnl/include/internal/`

Reply:
I added it and placed it here because I wanted it to take less time to recompile when it changed. Perhaps in the future, when there are no frequent changes to this file, the data from it can be combined with ntoskrnl\include\internal\*.h.